test/scrabble_test.rb

```diff
@@ -5,9 +5,35 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def test_it_exists
+    scrabble = Scrabble.new("a")
+    assert_instance_of Scrabble, scrabble
+  end
+
+  def test_it_can_score_a_single_letter
+    scrabble = Scrabble.new("a")
+    scrabble.point_values
```

Comment:
This line returns out the point_values hash, but you don't do anything with it afterwards.